js_task1/firstTask.js

```diff
@@ -0,0 +1,26 @@
+const input = document.querySelector(".first-task input");
+const pushButton = document.querySelector(".first-task button");
+const outArea = document.querySelector(".first-task .out-area");
+const symbol = " $ "; // output symbol
+
+function outputLine(symbolCount) {
+  outArea.innerHTML += `${symbol.repeat(symbolCount)} <br>`;
+}
+
+pushButton.addEventListener("click", () => {
+  outArea.innerHTML = "";
+  let number = Math.round(Number(input.value));
+  if (isNaN(number) || number <= 0) {
+    alert("Incorrect value");
+  } else {
```

Comment:
else - is redundant here. You can write 'return' after  alert and skip 'else' operator.

Reply:
done